src/main/java/Duke.java

```diff
@@ -1,10 +1,76 @@
+import java.util.Scanner;
+
 public class Duke {
+    private static final Task[] tasks = new Task[100];
+    private static final String horizontalLine = "    ____________________________________________________________\n";
```

Comment:
Since this is a constant and you won't be modifying this later, it should be in all caps so can use "HORIZONTAL_LINE" instead.